_rules/heading-descriptive-b49b2e.md

```diff
@@ -219,6 +190,17 @@ Heading marked up with `h1` element that does not describe the topic or purpose
 
 #### Failed Example 5
 
+Heading marked up with `h1` element that describes the topic or purpose of the following [palpable content][]. The heading is [visible][], but is not [included in the accessibility tree][].
+
+```html
+<html lang="en">
+	<h1 aria-hidden="true">Opening Hours</h1>
+	<p>We are open Monday through Friday from 10 to 16</p>
+</html>
+```
```

Comment:
I can move this to Inapplicable when I get a moment. Thanks @Jym77

Reply:
I would suggest if we're going to have an inapplicable example of a heading with aria-hidden, make it an empty heading, so that there isn't a 1.3.1 violation.

Reply:
@WilcoFiers Agreed. I will make this inapplicable with the empty heading.

Reply:
The commit message should have read **"Passed example 7"** not **"Passed example 3"** @WilcoFiers